cli/src/cmd/cast/send.rs

```diff
@@ -109,19 +157,6 @@ impl SendTxArgs {
                 tx.nonce = Some(provider.get_transaction_count(from, None).await?);
```

Comment:
Answered in https://github.com/foundry-rs/foundry/pull/4874#discussion_r1185013668 — essentially want to keep these code paths simple instead of further abstracting them.

Reply:
+1